asciicast/player.go

```diff
@@ -18,14 +19,14 @@ func NewPlayer() Player {
 	return &AsciicastPlayer{Terminal: terminal.NewTerminal()}
 }
 
-func (r *AsciicastPlayer) Play(path string) error {
+func (r *AsciicastPlayer) Play(path string, maxWait float64) error {
 	asciicast, err := Load(path)
 	if err != nil {
 		return err
 	}
 
 	for _, frame := range asciicast.Stdout {
-		delay := time.Duration(float64(time.Second) * frame.Delay)
+		delay := time.Duration(float64(time.Second) * math.Min(maxWait, frame.Delay))
```

Comment:
If `maxWait` is 0 (default value of float64) then this will always `time.Sleep(0)`.